tests/javaBase fails unless `LANGUAGE=en_US:en`

This fails on my machine even if I do not set `LANGUAGE`, probably since #3096 was merged.
```
 > cd build/tests/javaBase/
 > LANGUAGE=de_DE:de make jvm
../check_simple_example_jvm.sh "../../bin/fz -modules=java.base" javaHello.fz || exit 1
RUN javaHello.fz 17c17
< error: not_a_valid_file (No such file or directory)
---
> error: not_a_valid_file (Datei oder Verzeichnis nicht gefunden)
*** FAILED out on javaHello.fz
make: *** [../simple.mk:51: jvm] Fehler 1
```

--- a/tests/javaBase/Makefile
+++ b/tests/javaBase/Makefile
@@ -28,4 +28,6 @@ export LD_LIBRARY_PATH ?= $(JAVA_HOME)/lib/server
 # fix libjvm.dylib not found: see also https://stackoverflow.com/a/3172515
 export DYLD_FALLBACK_LIBRARY_PATH ?= $(JAVA_HOME)/lib/server
 FUZION_OPTIONS = -modules=java.base
+# needed for the exception text: see #3106
+export LANGUAGE = en_US:en
 include ../simple.mk

